package.json

```diff
@@ -264,7 +269,9 @@
                         "host": "${promptForHost}",
                         "password": "${promptForPassword}",
                         "rootDir": "${workspaceFolder}",
-                        "enableDebuggerAutoRecovery": false
+                        "enableDebuggerAutoRecovery": false,
+                        "enableAutoTerminate": false,
```

Comment:
@georgejecook I think this is the OLD name for `stopDebuggerOnAppExit`

Reply:
good catch - merge issue. thx